use_case_examples/cifar/README.md

```diff
@@ -0,0 +1,55 @@
+# CIFAR-10 and CIFAR-100 Classification with FHE
+
+This repository provides resources and documentation on different use-cases for classifying CIFAR-10 and CIFAR-100 images using Fully Homomorphic Encryption (FHE). Each use-case demonstrates different techniques and adaptations to work within the constraints of FHE.
```

Comment:
I think we need to state more the differences between notebooks:

- fine-tuning from a float32 network (can only be done with the same architecture)
- same accuracy as fine-tuned network but with a smaller model
- accelerating the cifar10 classification with the small network by splitting it between client/server

Something along those lines so we understand why we should look at one nb or the other

Reply:
You mean use cases instead of notebook right? I can add a sentence yes but we have a section for each use case so I wouldn't write too much detail in the description.

Reply:
I think the summary should explain why one would look at one use case or another

Reply:
Alright I added something.